src/api.js

```diff
@@ -2351,7 +2351,7 @@ api.declare({
   query: {
     continuationToken: /./,
     limit: /^[0-9]+$/,
-    disabled: /^(true|false)$/,
+    quarantineUntil: /\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d:[0-5]\d\.\d+([+-][0-2]\d:[0-5]\d|Z)/,
```

Comment:
Might it not be safer to say `quarantined: true|false`

Timestamps are complicated... and this is unlikely to match timestamps produced in python and other languages..
I'm pretty sure we use "ISO 8601" with "Z" for UTC time... but it gets complicated when some timestamps have microseconds and what not... Are you sure we covered all that here?

Note: I think `quarantined: true|false` is complicated enough. We can always add more advanced filters later.

Reply:
@jonasfj This will cover `ISO 8601` (e.g., `new Date().toISOString()`). Do you still want me to change `quarantineUntil` to `quarantined`? Also, do you only want that changed for `listWorkers()` only or for all endpoints returning `qurantineUntil`?

Reply:
I think it's the parameter in particular Jonas is concerned about.  We don't typically offer very sophisticated filtering arguments on our listXxx methods, so this seems odd.  And it will reject ISO 8601 timestamps that fall exactly on the second, as those do not have a `.` in them.

Reply:
My last commit changes quarantineUntil to quarantined :)